### DISCUSSION TITLE
Get the secret storage error

### DISCUSSION THREAD

**Discussion by RaptaG** [tutao/tutanota#6962] [Bugs]
I installed the Tuta Flatpak from Flathub on my fresh Fedora 40 KDE installation and after a few reboots the issue arised.

I read it was resolved since the library needed was added as a dependency..

Note: I have uninstalled KWallet

Can anybody help me fix it?

**Comment by charlag** [comment]
Hi
You need secret storage/keychain as part of your OS, it can be provided by KWallet or by gnome-keyring-daemon or by something else

**Reply by RaptaG** [reply]
I use the Flatpak version, isn't it built-in?

**Reply by charlag** [reply]
There's frontend and backend part of it. One is part of the app, another is part of the system.
